tasks/terraform-cli/src/commands/tf-workspace-new.ts

```diff
@@ -17,6 +17,13 @@ export class TerraformWorkspaceNew implements ICommand {
 
         const result = await this.runner.exec(options);
 
-        return result.toCommandResponse();
+
+        const command_result = result.toCommandResponse();
+        const skipExistingWorkspace = ctx.skipExistingWorkspace && command_result.message && command_result.message.indexOf("already exists") >= 0;
+        return new CommandResponse(
+            skipExistingWorkspace ? CommandStatus.Success : command_result.status,
+            command_result.message,
+            skipExistingWorkspace ? 1 : command_result.lastExitCode,

```

Comment:
```suggestion
            skipExistingWorkspace ? 0 : command_result.lastExitCode,
```
This should return 0 for success. 1 = fail